google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/cloud/executor/AppEngineExecutor.java

```diff
@@ -30,7 +39,15 @@ public AppEngineExecutor(AppEngineTask task) {
 
   @Override
   public void run() {
-    task.execute(this::setProcess);
+    task.execute(
+        process -> {
+          AppEngineExecutor.this.process = process;
```

Comment:
isn't `this.process` sufficient?

Reply:
it is, probably leftover from anonymous class.